common/src/View/CompilationRunner.cpp

```diff
@@ -82,8 +83,11 @@ namespace TrenchBroom {
                             IO::Disk::createDirectory(directoryPath);
                         }
 
+                        std::shared_ptr<IO::MapExportOptions> options = std::make_shared<IO::MapExportOptions>();
```

Comment:
Why is this modeled using `shared_ptr` in the first place? There is no shared ownership as far as I can see. I would use `unique_ptr` -- however, as I said above, I'd prefer you don't use inheritance for the export options, which would just allow you to use stack values.